pre-docs/plan_functions.md

```diff
@@ -200,15 +200,15 @@ puppetdb_fact(['app.example.com', 'db.example.com'])
 
 ### puppetdb\_query
 
-Makes a query to puppetdb using Bolt's PuppetDB client.
+Makes a query to [puppetdb](https://puppet.com/docs/puppetdb/latest/index.html) using Bolt's PuppetDB client.
```

Comment:
Is this generated by strings?

Reply:
https://github.com/puppetlabs/bolt/blob/master/bolt-modules/boltlib/lib/puppet/functions/puppetdb_query.rb#L5